src/Domain/MultilingualLink.php

```diff
@@ -92,11 +111,15 @@ public static function empty(): self
      */
     public static function fromArray(array $data): self
     {
+        /** @var MultilingualLink|null $parentData */
+        $parentData = $data[self::PARENT] ?? null;
+
         return new self(
             $data[self::ID],
             MultilingualString::fromArray($data[self::NAME]),
             MultilingualString::fromArray($data[self::LINK_TEXT]),
             MultilingualString::fromArray($data[self::LINK_URL]),
+            !empty($parentData) ? self::fromArray($parentData) : null,
```

Comment:
Same as above

Reply:
The same:

https://github.com/RRZE-Webteam/FAU-Studium-Common/blob/f9b1698f53be1d9882d6b9c9cdd4ccd7075cc2c2/src/Domain/MultilingualLink.php#L7-L16